benchmark.py

```diff
@@ -367,7 +397,21 @@ def _step(detail=False):
         for _ in range(self.num_warm_iter):
             _step()
 
-        t_run_start = self.time_fn()
+        if self.cuda_graph:
```

Comment:
CUDA Graphs can be helpful in training, but since larger batches are often used there isn't typically a perf gain from using them. @rwightman do you think we should just leave cuda graphs for inference but not training? I don't suspect above something like batch size 8 you'd see much of a tangible benefit.